evm/box/package.json

```diff
@@ -13,21 +13,21 @@
     "format": "prettier --write \"**/*\"",
     "migrate:dev": "npx truffle migrate --reset --network cldev",
     "migrate:live": "npx truffle migrate --network live",
-    "setup": "echo \"No setup required for @chainlink/box\"",
     "test": "npx truffle test"
   },
   "license": "MIT",
   "dependencies": {
-    "chainlink": "0.6.5",
-    "chainlink-test-helpers": "0.6.0-6",
+    "chainlink": "0.7.5",
     "link_token": "^1.0.6",
     "openzeppelin-solidity": "^1.12.0"
   },
   "devDependencies": {
     "@chainlink/eslint-config": "0.0.1",
     "@chainlink/prettier-config": "0.0.1",
+    "chai": "^4.2.0",
     "depcheck": "^0.8.3",
     "eslint": "^6.3.0",
+    "openzeppelin-test-helpers": "^0.4.3",
```

Comment:
Is there a reason why we're adding this? `openzeppelin-test-helpers` only supports `web3` or `truffle` env when we're going to be using ethers instead, so this will have to be ripped out once we convert these tests to typescript + ethers

Reply:
The box is a user-facing project. It's separate from our recent changes to using Ethers.